### ISSUE TITLE
Question about the Discrete Variables Calculation 

### ISSUE THREAD

**Issue by smiles724** [jasonkyuyim/multiflow#6]
Hi thanks for contributing to this wonderful work in protein co-design, particularly in the novel CTMC mechanism. The code is clean and easy to understand. However, I am little confused about the implementation of the discrete flow matching part.

Specifically, in the `FlowModel` module, you directly forward the node embeddings into a linear layer for the amino acid type logit prediction. Then a cross entropy loss is followed to compute the amino acid type loss. 
![image](https://github.com/user-attachments/assets/60c75c88-22b4-406e-83e0-0af78ba6ae05)
![image](https://github.com/user-attachments/assets/786496bd-9a4f-4377-ad8a-05c90a6f67fb)

However, in the paper present, you proposed to maximize the conditional log-likelihood of the residue types instead of the simple CE loss. Can you please give an explanation for this nuance? Thanks.
![image](https://github.com/user-attachments/assets/a785a162-86da-4217-828d-92c53ec9c8e5)

**Comment by jasonkyuyim** [comment]
I'm not sure I follow the question. Minimizing the cross entropy loss is the same as maximizing the log-likelihood right?

**Comment by smiles724** [comment]
Yes. I misunderstood the formula at that time.
